### PR TITLE
feat: allows to specify id for put method

### PR DESCRIPTION
**Pull request by maasencioh**

### CODE REVIEW

File: src/DataDrive.ts [hunk 1]
```diff
@@ -90,8 +90,8 @@ export class DataDrive {
   public async put(
     filename: string,
     content: Buffer | NodeJS.ReadableStream | string,
+    id: string = randomUUID(),
```
Comment (targos) [inline src/DataDrive.ts]: can you make it an options object? I don't want to do a breaking change if we add other options later

Comment (maasencioh) [reply src/DataDrive.ts]: good idea, thanks!

**Comment by targos** [comment]
It's a bit annoying but for now you need to duplicate the types in https://github.com/zakodium/adonis-datadrive/blob/c95f6cd2cb98fbabfb180fce6078874c15912f10/src/types.ts#L55-L58
`types.ts` contains the definitions that you will use in your project.

**Comment by maasencioh** [comment]
Just curious, why you have it like that?

**Comment by targos** [comment]
We have to define the types in the IoC namespace, because that's how it will be imported in the apps

image

There's a way to limit the duplication and I'm starting to use it in adonis-mongodb, but it cannot be avoided entirely.

**Pull request merged**